src/Rector/Class_/CommandPropertyToAttributeRector.php

```diff
@@ -118,34 +206,104 @@ public function refactor(Node $node): ?Node
             }
         }
 
-        $node->attrGroups[] = $this->createAttributeGroup($defaultName, $defaultDescription);
-
-        return $node;
+        return $defaultDescription;
     }
 
-    private function createAttributeGroup(string $defaultName, ?string $defaultDescription): AttributeGroup
+    private function resolveAliases(Class_ $class): ?Array_
     {
-        $attributeGroup = $this->phpAttributeGroupFactory->createFromClass(self::ATTRIBUTE);
+        $commandAliases = null;
+        $classMethod = $class->getMethod('configure');
+        if (! $classMethod instanceof ClassMethod) {
+            return null;
+        }
+        $this->traverseNodesWithCallable($class->stmts, function (Node $node) use (&$commandAliases) {
```

Comment:
Oh, it need to be `$classMethod->stmts`:

```suggestion
        $this->traverseNodesWithCallable((array) $classMethod->stmts, function (Node $node) use (&$commandAliases) {
```

Reply:
++